packages/clerk-js/src/ui/utils/errorHandler.ts

```diff
@@ -15,10 +15,17 @@ function setFieldErrors(fieldStates: Array<FormControlState<string>>, errors: Cl
   }
 
   fieldStates.forEach(field => {
-    const error = errors.find(err => {
+    let errorCallback = field?.callback;
+
+    if (!errorCallback) {
+      errorCallback = (err: ClerkAPIError[]) => err[0];
```

Comment:
❓ Should we return a string in this function here? I think it's a bit confusing to return either a `ClerkAPIError` object or a string. Is this handled appropriately?

Reply:
@anagstef No need to do this here. This is handled later on from `translateError` in useLocalization([here](https://github.com/clerkinc/javascript/blob/d0816ee1385ae942b9853d617ce5d2e4ad6f5cd1/packages/clerk-js/src/ui/localization/makeLocalizable.tsx#L73))